docs/users.rst

```diff
@@ -1170,3 +1170,95 @@ stored in koji build, in section ``build.extra.operator_manifests.appregistry``.
 
 Manifests will not be pushed to the application registry for scratch builds,
 isolated builds, or re-builds to prevent unwanted changes
+
+
+Building Source Container Images
+================================
+
+OSBS is able to build source container image from a particular koji build
+previously created by OSBS. To create a source container build you have to
+specify either koji N-V-R or build ID for the image build you want to
+create a source container image for.
+
+
+Under the hood the BSI_ project is used to generate source images from
+sources identified and collected by OSBS.
+
```

Comment:
Do we want to mention that the BSI script should be available as `bsi` in the buildroot $PATH?

Reply:
added